php/meterpreter/meterpreter.php

```diff
@@ -42,8 +42,27 @@ function register_command($c, $i) {
   }
 }
 
+# Debugging payload definitions
+define("DEBUGGING", false);
+define("DEBUG_LOGFILE_PATH", "");
+
+function my_logfile($str) {
+  if (DEBUGGING AND DEBUG_LOGFILE_PATH != "") {
+    if (!isset($GLOBALS['logfile'])) {
+      $GLOBALS['logfile'] = fopen(DEBUG_LOGFILE_PATH, 'a');
```

Comment:
If `DEBUG_LOGFILE_PATH` is `""` by default, will this implementation raise warnings for every log attempt? 👀 

```
php > $y = fopen("", "rb");

Warning: fopen(): Filename cannot be empty in php shell code on line 1
php > $y;
php > var_dump($y);
bool(false)
```